explorer/explorer.go

```diff
@@ -659,6 +260,10 @@ func New(dataSource explorerDataSourceLite, primaryDataSource explorerDataSource
 		"int64Comma": func(v int64) string {
 			return humanize.Comma(v)
 		},
+		"ticketWindowProgress": func(i int) float64 {
+			p := (int64(i) / exp.ChainParams.StakeDiffWindowSize) * 100
```

Comment:
The conversions here are dubious.  Why divide in integer, (and does it produce the right result)?